### PR TITLE
Update the infrastructure issue template

### PR DESCRIPTION
**Pull request by ricardoamador**
*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/6_infrastructure.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        Thank you for using Flutter!
+
+        It looks like you found an issue with our Infrastructure services.
+        Please complete the form below so that we can help to resolve your
+        issue quickly.
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      options:
+      - label: I have searched the [existing issues](https://github.com/flutter/flutter/issues)
```
Comment (keyonghan) [inline .github/ISSUE_TEMPLATE/6_infrastructure.yml:19]: Nit: https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3Ateam-infra

File: .github/ISSUE_TEMPLATE/6_infrastructure.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        Step 1:
+        Step 2:
+        ..
+        Step n:
+  - type: textarea
+    attributes:
+      label: Expected results
+      description: If you have a bug, What should the expect output be?
+      value: |
+        I expect to see X when Y is finished.
```
Comment (keyonghan) [inline .github/ISSUE_TEMPLATE/6_infrastructure.yml:69]: Nit: extra line

**Review by keyonghan — COMMENTED** [review]
It shows some errors:
https://github.com/flutter/flutter/blob/b23ba6164b4c41d6a07d311856e37ba1844cf346/.github/ISSUE_TEMPLATE/6_infrastructure.yml
Screenshot 2023-08-28 at 7 12 43 PM

File: .github/ISSUE_TEMPLATE/6_infrastructure.yml (hunk 1 above)
Comment (ricardoamador) [reply .github/ISSUE_TEMPLATE/6_infrastructure.yml:19]: Ah good catch!

File: .github/ISSUE_TEMPLATE/6_infrastructure.yml (hunk 2 above)
Comment (ricardoamador) [reply .github/ISSUE_TEMPLATE/6_infrastructure.yml]: Added!

File: .github/ISSUE_TEMPLATE/6_infrastructure.yml [hunk 3]
```diff
@@ -11,12 +11,12 @@ body:
...
         It looks like you found an issue with our Infrastructure services.
         Please complete the form below so that we can help to resolve your
-        issue quickly.
+        issue as quickly as possible.
   - type: checkboxes
     attributes:
       label: Is there an existing issue for this?
       options:
-      - label: I have searched the [existing issues](https://github.com/flutter/flutter/issues)
+      - label: I have searched the [existing infra issues]( https://github.com/flutter/flutter/issues?q=is%3Aopen+is%3Aissue+label%3Ateam-infra)
```
Comment (keyonghan) [inline .github/ISSUE_TEMPLATE/6_infrastructure.yml:19]: The extra space needs to be removed.

**Comment by keyonghan** [comment]
> I believe I have fixed the errors though I could not find a way to render the page to test it. For some reason the github preview just shows me the yml file again and not the rendered form.

You can view the rendered form here: https://github.com/flutter/flutter/blob/f862d7ec34753f9047105b0d43a22e9d91a15f71/.github/ISSUE_TEMPLATE/6_infrastructure.yml

**Comment by ricardoamador** [comment]
Eww! That does not look good. Okay let me update it.

**Comment by keyonghan** [comment]
Seems another error with the latest commit:
https://github.com/flutter/flutter/blob/c161de21bf2250e7177848ed3fd12e7604f7ca37/.github/ISSUE_TEMPLATE/6_infrastructure.yml

Screenshot 2023-08-29 at 1 22 17 PM

File: .github/ISSUE_TEMPLATE/6_infrastructure.yml (hunk 3 above)
Comment (ricardoamador) [reply .github/ISSUE_TEMPLATE/6_infrastructure.yml]: Removed.

**Review by keyonghan — APPROVED** [review]
LGTM

**Pull request merged**